docker/generate_makefile.py

```diff
@@ -58,116 +62,120 @@ def _print_makefile_run_template(image):
 \t-e FUZZER={fuzzer} \\\n\
 \t-e BENCHMARK={benchmark} \\\n\
 \t-e FUZZ_TARGET=$({benchmark}-fuzz-target) \\\
-'.format(fuzzer=fuzzer, benchmark=benchmark))
+\n'
 
-        params = ""
         if run_type == 'test-run':
-            print('\t-e MAX_TOTAL_TIME=20 \\\n\t-e SNAPSHOT_PERIOD=10 \\')
+            section += '\t-e MAX_TOTAL_TIME=20 \\\n\t-e SNAPSHOT_PERIOD=10 \\\n'
         if run_type == 'debug':
-            print('\t--entrypoint "/bin/bash" \\\n\t-it ', end='')
+            section += '\t--entrypoint "/bin/bash" \\\n\t-it '
         elif run_type == 'repro-bugs':
-            print('\t-v {path}:/testcases \\\n\
-\t--entrypoint /bin/bash '.format(path=bugs_testcases_dir),
-                  end='')
-            params = " -c \"for f in /testcases/*; do\
-                    echo _________________________________________;\
-                    echo \\$$f:;\
-                    \\$$OUT/\\$$FUZZ_TARGET -timeout=25\
-                    -rss_limit_mb=2560 \\$$f; done;\""
-
+            section += f'\t-v {testcases_dir}:/testcases \\\n\t'
+            section += '--entrypoint /bin/bash '
+            section += os.path.join(BASE_TAG, image['tag'])
+            section += ' -c "for f in /testcases/*; do '
```

Comment:
Did you verify this works, i wonder why it was added after the set of ifs before.

Reply:
Yep I verified this.